lib/screens/stop_watch.dart

```diff
@@ -0,0 +1,301 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_internship/helpers/my_app1_icons.dart';
+
+class StopWatch extends StatefulWidget {
+  const StopWatch({super.key});
+
+  @override
+  State<StopWatch> createState() => _StopWatchState();
+}
+
+class _StopWatchState extends State<StopWatch> {
+  int hundredths = 0, seconds = 0, minutes = 0;
+  String digitHundredths = '00', digitSeconds = '00', digitMinutes = '00';
```

Comment:
I think that you do not need String representation of hundredths, seconds, minutes